From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Jeremy Apthorp <nornagon@nornagon.net>
Date: Thu, 20 Sep 2018 17:48:38 -0700
Subject: gtk_visibility.patch

Allow electron and brightray to depend on GTK in the GN build.

diff --git a/build/config/linux/gtk/BUILD.gn b/build/config/linux/gtk/BUILD.gn
index deae4d3455a8b54292dd3abc0a316f9ca60b4104..fd5d906b98b047b245824a4b3d6f0c528c96ee1b 100644
--- a/build/config/linux/gtk/BUILD.gn
+++ b/build/config/linux/gtk/BUILD.gn
@@ -18,6 +18,8 @@ group("gtk") {
   visibility = [
     "//chrome/test:interactive_ui_tests",
     "//chrome/test:unit_tests",
+    "//electron:*",
+    "//electron/brightray:*",
     "//examples:peerconnection_client",
     "//gpu/gles2_conform_support:gles2_conform_test_windowless",
     "//remoting/host",
